static/locales/en-US.yaml

```diff
@@ -119,7 +120,7 @@ User Playlists:
     it listed here
   Empty Search Message: There are no videos in this playlist that matches your search
   Search bar placeholder: Search in Playlist
-History:
+History Page:
```

Comment:
Why rename this?
`$t("History.History")` would have gotten the history value before the update.

Also all locales that already have the History translated will need to update to "History Page" in order to not use the english translations

Reply:
According to the comment `# Webkit Menu Bar` those entries are for `Webkit Menu Bar` and seems that they can only be placed on top level
I have no idea if that's true or not
Let me know?

The commit for that comment and many other entries are from [a commit 2 years ago](https://github.com/FreeTubeApp/FreeTube/commit/8aa0dcd243cf09312a25559b6be59ab71ee5895b)

Reply:
At the moment the menu doesn't get translated at all, so the translation change is not necessary yet, until someone decides to setup translations for the menu.

Reply:
OK I will revert the change
Confusing!